Formula/go.rb

```diff
@@ -4,9 +4,9 @@ class Go < Formula
   license "BSD-3-Clause"
 
   stable do
-    url "https://golang.org/dl/go1.15.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.15.src.tar.gz"
-    sha256 "69438f7ed4f532154ffaf878f3dfd83747e7a00b70b3556eddabf7aaee28ac3a"
+    url "https://dl.google.com/go/go1.15.1.src.tar.gz"
```

Comment:
```suggestion
    url "https://golang.org/dl/go1.15.1.src.tar.gz"
```

Reply:
`https://dl.google.com/go/go1.15.src.tar.gz` is the true location.

```
$ curl -I https://golang.org/dl/go1.15.src.tar.gz
HTTP/1.1 302 Found
Date: Tue, 01 Sep 2020 19:23:36 GMT
Content-Type: text/html; charset=utf-8
Location: https://dl.google.com/go/go1.15.src.tar.gz
```

Reply:
Similar work that gradle team has done, https://github.com/gradle/gradle/pull/2152

Reply:
Revert the style change, we can talk about it in a separate thread.